nsdb-cli/src/main/scala/io/radicalbit/nsdb/cli/NsdbRestore.scala

```diff
@@ -43,22 +42,17 @@ object NsdbRestore extends App {
     } text "the remote port"
     opt[String]("path").required() action { (x, c) =>
       c.copy(sourcePath = x)
-    } text "path of the file to restore"
+    } text "path of the metadata folder to restore"
   }
 
   parser.parse(args, Params(None, None, "")) foreach { params =>
     import scala.concurrent.duration._
 
     val clientGrpc = new GRPCClient(host = params.host.getOrElse("127.0.0.1"), port = params.port.getOrElse(7817))
 
-    Await.ready(clientGrpc.checkConnection(), 5.seconds).value.get match {
-      case Success(_) =>
-        val response: RestoreResponse = Await.result(clientGrpc.restore(RestoreRequest(params.sourcePath)), 10.seconds)
-        if (response.startedSuccessfully) println("Restore process started successfully")
-        else sys.error(response.errorMsg)
-      case Failure(_) => sys.error(s"instance is not available at the moment.")
-    }
-
+    val response = Await.result(clientGrpc.restore(RestoreRequest(params.sourcePath)), 10.seconds)
```

Comment:
Might the timeout be configurable?

Reply:
Good point. Since this is a client application (will be used to generate a `nsdb-restore.sh` script, we haven't figure out a proper way to make timestamps configurable. It might be a good scope for a future PR. what's your thought ?